Google.Api.Gax.Grpc/GrpcChannelOptions.cs

```diff
@@ -201,6 +221,7 @@ IReadOnlyList<CustomOption> MergeCustomOptions(IReadOnlyList<CustomOption> origi
             other is object &&
             EnableServiceConfigResolution == other.EnableServiceConfigResolution &&
             KeepAliveTime == other.KeepAliveTime &&
+            KeepAliveTimeout == other.KeepAliveTimeout &&
```

Comment:
Should probably be added to `GetHashCode` as well.
And there's an Equality test where this should be added as well.

Reply:
Ah, well done - thanks for spotting that. Will fix now in a separate commit for you to check, then will merge those two together.